src/Worker.Extensions.DurableTask/FunctionsOrchestrator.cs

```diff
@@ -58,14 +58,6 @@ internal class FunctionsOrchestrator : ITaskOrchestrator
         FunctionsOrchestrationContext orchestrationContext)
     {
         Task orchestratorTask = next(functionContext);
-        if (!orchestratorTask.IsCompleted && !orchestrationContext.IsAccessed)
```

Comment:
I am not entirely sure what problems this was supposed to catch, so I can't really tell if there is something that is no longer being checked. Based on your comments earlier I suppose this check did not have a well-defined function.

Reply:
Yeah I want to remove it, but don't want to block this fix on that discussion. So I added it back in, we can discuss that later.